bot/code_review_bot/report/base.py

```diff
@@ -101,6 +101,10 @@ def stats(analyzer, items):
             if analyzer.startswith("source-test-"):
                 analyzer = analyzer[12:]
 
+            # Nicer name for code coverage
+            elif analyzer == "coverage":
+                analyzer = "code coverage analysis"
```

Comment:
Nit: I wish we had a "nicer" way to handle nicification of "source-test-" and "coverage"

Reply:
Followup in https://github.com/mozilla/code-review/issues/554 (it was part of the plan to output nicer names for the lint results headers)